src/scripts.js

```diff
@@ -176,25 +178,30 @@ function viewRecDetails (card) {
         <div class="instructions">
           ${populateInstructions(card)}
         </div>
-      </article>
-    </section>`;
+      </article>`;
+}
 
-  // populateIngredients(card);
+function populateCost (baseRecipe) {
+  const totalCost = baseRecipe.calcCost(newRepository.ingredientIndex)
+  return totalCost;
 }
 
 function populateIngredients (baseRecipe) {
-  const names = baseRecipe.ingNames(newRepository.ingredientIndex)
-  names.reduce((ingList, ing) => {
-    ingredients += `<p class="tag">#${tag}</p>\n`
-    return tagList
-  },[])
+  const ing = baseRecipe.ingNames(newRepository.ingredientIndex)
+  let result = '';
+  let index = 0;
+  console.log(index)
+  while(index < ing.length){
+    result += `<p>- ${ing[index]}: ${baseRecipe.ingredients[index].quantity.amount} ${baseRecipe.ingredients[index].quantity.unit}</p>\n`
```

Comment:
Glad we were able to figure this out, considering we were able to get rid of the undefined this would be fine to replace the while loop with a do while loop as well